src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -415,10 +411,71 @@ function createModel(program: Program, generateConvenienceAPI: boolean = false):
         }
     }
 
+    function emitClient(
+        client: Client | OperationGroup,
+        parent?: Client
+    ): InputClient {
+        const operations = listOperationsInOperationGroup(program, client);
+        let clientDesc = "";
+        if (operations.length > 0) {
+            const container = ignoreDiagnostics(
+                getHttpOperation(program, operations[0])
+            ).container;
+            clientDesc = getDoc(program, container) ?? "";
+        }
+
+        const inputClient = {
+            Name: client.kind === ClientKind.DpgClient ? client.name : client.type.name,
+            Description: clientDesc,
+            Operations: [],
+            Protocol: {},
+            Creatable: client.kind === ClientKind.DpgClient,
+            Parent: parent?.name
+        } as InputClient;
+        for (const op of operations) {
+            if (lroMonitorOperations.has(op)) continue;
```

Comment:
Then, where shall we put lroMonitorOperations?

Reply:
At the first it will calculate all the LroMonitorOperations. And if it is LroMonitorOperation, we will not generate it.